### PR TITLE
Create culture-and-values.md

### PR DESCRIPTION
**Pull request by AlaskaCasey**

### CODE REVIEW

**Comment by AlaskaCasey** [comment]
@serverless/team please review and add any comments/ suggestions.

File: culture-and-values.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Our culture##
```
Comment (mthenw) [inline culture-and-values.md]: "##"

File: culture-and-values.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+[TBD] 
+
+## Values##
```
Comment (mthenw) [inline culture-and-values.md]: "##"

File: culture-and-values.md (hunk 1 above)
Comment (pmuens) [inline culture-and-values.md]: So `## Our culture##` --> `## Our culture` (`##` means that the text after it should be transformed to a heading with a depth of 2)

File: culture-and-values.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+**Blaze the trail**
+
+As the first mover in the serverless space our team is bold and ambitious. We think outside the box and are always looking for ways to embrace new technology and make it accessible to our community.
+
+**Open source forever**
+
+The open source community is an integral strategy to our success. We will never forget where we came from and pledge to always give back.
+
+**Simplicity is king**
```
Comment (eahefnawy) [inline culture-and-values.md]: 👍

**Comment by eahefnawy** [comment]
Thanks @AlaskaCasey ... I think being a remote/distributed team/company is part of our culture, no? 🤔 ... Anyone who joins the team should be comfortable with working remotely or at least communicating with a remote team.

**Pull request merged**